source/documentation/migration-engine-3.6-to-4.0.html.md

```diff
@@ -34,7 +34,7 @@ Install `ovirt-engine` package and dependencies:
 
 Copy your backup to the new system, and restore it with following command:
 
-    # engine-backup --mode=restore --no-restore-permissions --provision-db --file=engine-backup.tar.gz --log=engine-backup-restore.log
+    # engine-backup --mode=restore --no-restore-permissions --provision-db --provision-dwh-db --file=engine-backup.tar.gz --log=engine-backup-restore.log
```

Comment:
You might suggest to add this option only if dwh is set up in the 3.6 machine. Thanks.